src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2652,23 +2654,31 @@ The Signal API is available inside ProActive Tasks through the *signalapi* <<_gl
 
 It is easy to use, and offers a set of basic operations on signals summarized as follows:
 
- * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals.
+ * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals with or without variables.
    This operation constructs new signals (by adding the prefix `ready_` to the input signals), then adds them to job signals.
    For example, if a job declares ready for the signal `soft_stop`, the signal `ready_soft_stop` is added to the set of job signals.
    Additionally, if the input signal (e.g., `soft_stop`) already exists among the job signals (which means that it has been sent and handled previously),
     the operation ready for signal automatically removes it, thus the job is ready again to receive and handle the same signal.
 
- * *sendSignal*, *sendManySignals*: adds one or more signals given as input to the current job signals.
+ * *sendSignal*, *sendManySignals*: adds one or more signals with or without variables given as input to the current job signals.
    This operation removes the corresponding ready signals if they already exist. For instance, sending `soft_stop` removes the signal
    `ready_soft_stop` if it already exists among the job signals.
 
- * *checkForSignals*: verifies whether one or more signals exist among the set of job signals.
+ * *checkForSignals*: verifies whether all of the signals given as input exist among the set of job signals.
 
  * *waitFor*, *waitForAny*: blocking instruction that waits until one signal (among those given as input) is added to the set of job signals.
```

Comment:
you should describe briefly the return type in case of a signal with variables (with a reference to the javadoc)